ffai/ai/layers.py

```diff
@@ -592,8 +581,27 @@ def produce(self, game):
             return out
         for player in active_team.players + game.get_opp_team(active_team).players:
             if player.position is not None:
-                out[player.position.y][player.position.x] = (player.get_ma() - player.state.moves) * 0.1
+                out[player.position.y][player.position.x] = max(0, (player.get_ma() - player.state.moves) * 0.1)
         return out
 
     def name(self):
         return "movement left"
+
+
+class GFIsLeftLayer(FeatureLayer):
+
+    def produce(self, game):
+        out = np.zeros((game.arena.height, game.arena.width))
+        active_team = game.state.available_actions[0].team if len(game.state.available_actions) > 0 else None
+        if active_team is None:
+            return out
+        for player in active_team.players + game.get_opp_team(active_team).players:
+            if player.position is not None:
+                has_sprint = player.has_skill(Skill.SPRINT)
```

Comment:
Rewrite as: num_max_gfi = 2 if... else 3. To make be able to write next line without if statements.